### PR TITLE
AP_Motors: separate roll and pitch limits

### PR DESCRIPTION
**Pull request by lthall**
This pull request separates the limits for roll, pitch. Currently if one is limited both PID loops are limited. This fixes this problem.

### CODE REVIEW

**Comment by tridge** [comment]
@lthall did you mean to include the acro sqrt_controller changes?

**Comment by lthall** [comment]
I must have based in on my other pull request instead of master!

Thanks!!

File: libraries/AP_Motors/AP_Motors6DOF.cpp [hunk 1]
```diff
@@ -407,7 +409,8 @@ void AP_Motors6DOF::output_armed_stabilizing_vectored()
     float linear_out[AP_MOTORS_MAX_NUM_MOTORS]; // 3 linear DOF mix for each motor
 
     // initialize limits flags
-    limit.roll_pitch = false;
+    limit.pitch = false;
```
Comment (OXINARF) [inline libraries/AP_Motors/AP_Motors6DOF.cpp]: pitch -> roll

Comment (lthall) [reply libraries/AP_Motors/AP_Motors6DOF.cpp]: I hate being dyslexic some time!!! I went through and checked each one of those TWICE!

Comment (lthall) [reply libraries/AP_Motors/AP_Motors6DOF.cpp]: Thanks mate!!! Fixed

**Comment by OXINARF** [comment]
Any objections on merging this? @tridge @rmackay9?

**Comment by rmackay9** [comment]
@OXINARF, I'd like to have another look at this before it goes in

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**